src/views/Dashboard/Dashboard.tsx

```diff
@@ -64,41 +39,69 @@ const DEXWidget: React.FC = () => {
 };
 
 const SendWidget: React.FC = () => {
-  const { accountRs } = useAccount();
-  const [toAddress, setToAddress] = useState<string>();
+  const { accountRs, publicKey } = useAccount();
```

Comment:
we might want to adopt a code standard where we always put the 'custom' hooks at the top or bottom of the block of useState hooks. at least i'd like to avoid the situation in this file currently where useAccount sandwhiches useState with useApi making it hard to see the useApi hook.

super small thing, possibly much more important as state gets larger for certain components.

Reply:
we can probably decide at some point and then just go through it all and fix it up then moving forward follow the convention